lib_eio/unix/fork_action.c

```diff
@@ -237,3 +237,23 @@ static void action_dups(int errors, value v_config) {
 CAMLprim value eio_unix_fork_dups(value v_unit) {
   return Val_fork_fn(action_dups);
 }
+
+static void action_setpgid(int errors, value v_config) {
+  #ifdef _WIN32
+  eio_unix_fork_error(errors, "set_cloexec", "Unsupported operation on windows");
```

Comment:
```suggestion
  eio_unix_fork_error(errors, "setpgid", "Unsupported operation on windows");
  _exit(1);
```

Perhaps `eio_unix_fork_error` should call `_exit` itself. Looks like everything that uses it either does that or is wrong.

Reply:
Indeed!